client/src/main/java/tds/assessment/Item.java

```diff
@@ -137,10 +141,47 @@ public void setItemProperties(List<ItemProperty> itemProperties) {
     }
 
     /**
+     * @return The full path and file name of the item's XML file
+     */
+    public String getItemFilePath() {
+        return itemFilePath;
+    }
+
+    public void setItemFilePath(String itemFilePath) {
+        this.itemFilePath = itemFilePath;
+    }
+
+    /**
+     * @return The full path and file name of the item's stimulus file (if there is one)
+     */
+    public String getStimulusFilePath() {
+        return stimulusFilePath;
+    }
+
+    public void setStimulusFilePath(String stimulusFilePath) {
+        if (stimulusFilePath == null) {
```

Comment:
This goes to the larger question if we should allow these to be nullable or not.  I think there are many fields in the client objects that are nullable.